doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -268,7 +268,7 @@ integration with 3rd party tools like logstash.
       #  server: 127.0.0.1
       #  port: 6379
       #  async: true ## if redis replies are read asynchronously
-      #  mode: list ## possible values: list (default), channel
+      #  mode: list ## possible values: list (default), rlist (uses RPUSH instead of LPUSH), channel
```

Comment:
@inliniac i did an additional look at this. I would suggest not just using 'lpush' and 'rpush' with backwards compatibility of 'list' i would also propose to use 'publish' for the 'channel' mode. That would be more consistent as 'publish' the redis command for channels like [lr]push are the commands for lists. 

I also noticed that currently we use channel whenever mode is configured to any value unequal to "list". I think we either should exit here with an error or log a warning and fallback to the default which would be "list".

What do you think?